### PR TITLE
Added streets to sample feed

### PR DESCRIPTION
**Pull request by pstenbjorn**
@jungshadow this PR is very narrow, I added some street_segment elements into the existing sample feed by @jdmgoogle.

### CODE REVIEW

**Comment by jdmgoogle** [comment]
LGTM

File: sample_feed.xml [hunk 1]
```diff
@@ -1739,4 +1739,128 @@
     <Number>0304</Number>
     <PollingLocationId>pl81213</PollingLocationId>
   </Precinct>
+    <StreetSegment id="ss300071">
```
Comment (jungshadow) [inline sample_feed.xml]: Would you mind fixing the indentation a bit?

File: sample_feed.xml [hunk 2]
```diff
@@ -1739,4 +1739,128 @@
...
+    <PrecinctId>pre92353</PrecinctId>
+    <StartHouseNumber>1</StartHouseNumber>
+    <EndHouseNumber>106</EndHouseNumber>
+  </StreetSegment>
+  <StreetSegment id="ss302215">
+    <NonHouseAddress>
+    <City>CHARLOTTESVILLE</City>
+    <State>VA</State>
+    <StreetName>BOARS HEAD</StreetName>
+    <StreetSuffix>LN</StreetSuffix>
```
Comment (jungshadow) [inline sample_feed.xml]: The last few of these have a `StreetSuffix`, but the others don't. Is that because of the data?

**Comment by jungshadow** [comment]
Just two comments. When they're cleaned-up/answered, you get a LGTM.

**Comment by jungshadow** [comment]
Related to #151.

**Comment by pstenbjorn** [comment]
@jungshadow I fixed the hanging indent and used a different subset of streets segments which are more representative (with suffixes).  Unless some one has an issue I will close this PR in about 10 minutes.

File: sample_feed.xml [hunk 3]
```diff
@@ -1739,4 +1739,134 @@
     <Number>0304</Number>
     <PollingLocationId>pl81213</PollingLocationId>
   </Precinct>
+  <StreetSegment id="ss309904">
+    <NonHouseAddress>
+    <City>GREENWOOD</City>
```
Comment (jungshadow) [inline sample_feed.xml]: `NonHouseAddress` subelements should also be indented.

**Comment by jungshadow** [comment]
Last nitpicky thing and then LGTM. After fix, merge when ready.

**Pull request merged**